dev/conductor/core/lib/src/globals.dart

```diff
@@ -14,13 +14,36 @@ const String kForceFlag = 'force';
 
 const List<String> kBaseReleaseChannels = <String>['stable', 'beta'];
 
-const List<String> kReleaseChannels = <String>[...kBaseReleaseChannels, FrameworkRepository.defaultBranch];
+const List<String> kReleaseChannels = <String>[
+  ...kBaseReleaseChannels,
+  FrameworkRepository.defaultBranch
+];
 
-const String kReleaseDocumentationUrl = 'https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process';
+const String kReleaseDocumentationUrl =
+    'https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process';
 
-const String kLuciPackagingConsoleLink = 'https://ci.chromium.org/p/flutter/g/packaging/console';
+const String kLuciPackagingConsoleLink =
+    'https://ci.chromium.org/p/flutter/g/packaging/console';
 
-const String kWebsiteReleasesUrl = 'https://docs.flutter.dev/development/tools/sdk/releases';
+const String kWebsiteReleasesUrl =
+    'https://docs.flutter.dev/development/tools/sdk/releases';
+
+const String discordReleaseChannel =
```

Comment:
Global consts should be prefixed with k

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#begin-global-constant-names-with-prefix-k

Reply:
This directly conflicts with the Dart style guide in Effective Dart. 

https://dart.dev/guides/language/effective-dart/style#dont-use-prefix-letters

As this is a Dart project, I went with their style guide.

Reply:
Since this is in the Flutter repo, the Flutter repo style guide will supersede effective dart

Reply:
That is a fair point, I'll make the change.